### PR TITLE
Fixes some issues, speeds up deploy by doing local only, and removes …

### PR DESCRIPTION
**Pull request by treeder**
…one more step.

This requires `fn` cli version 0.3.77 or later.

### CODE REVIEW

File: services/Makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 deploy:
-	cd draw; fn deploy myapp
```
Comment (carimura) [inline services/Makefile]: I use this makefile quite often to push into remote registry... can you remove the --local? or create deploy-local?

File: vista.html [hunk 2]
```diff
@@ -1,317 +0,0 @@
-<!DOCTYPE html>
```
Comment (carimura) [inline vista.html]: could add vista.html to gitignore to avoid this.

**Comment by carimura** [comment]
If you feel ambitious, you can change S3_ to STORAGE_ globally. :) Owise it's on my short list of things to do.

File: vista.html (hunk 2 above)
Comment (treeder) [reply vista.html]: it already is: https://github.com/carimura/vista/blob/master/.gitignore

File: services/Makefile (hunk 1 above)
Comment (treeder) [reply services/Makefile]: Ok, if you pass do `./setup.sh local`, it will do local only.

**Pull request merged**